packages/contracts/networks.json

```diff
@@ -2,7 +2,8 @@
   "goerli": {
     "url": "https://goerli.infura.io/v3/92aa62d2bb5449cfafe04b83ca8636f1",
     "isTestnet": true,
-    "chainId": 5
+    "chainId": 5,
+    "gasPrice": 100000000000
```

Comment:
should we leave this here?
perhaps we can grab it from .env and add in hardhat.config.ts
just a suggestion, feel free to resolve this comment

Reply:
good finding. removed